spec/unit/server_spec.rb

```diff
@@ -191,6 +191,55 @@ def app
         end
       end
     end
+
+    context "broken description" do
+      before do
+        store_raw_description("foo", <<-EOT
+        {
```

Comment:
Hi @fschueller, thanks for reviewing! 

I didn't see how could I use the factory's `create_test_description` without having it [validating](https://github.com/SUSE/machinery/blob/master/spec/support/system_description_factory.rb#L87) the JSON. That's why I used `store_raw_description` instead. You can see that I've included the `foo` attribute which would make another error to be raised.

Or maybe I just missed something. ;)

P.S.: Maybe some sections of that broken JSON can be removed.